WalletWasabi/WabiSabi/Backend/Rounds/Arena.cs

```diff
@@ -246,7 +246,9 @@ private async Task StepOutputRegistrationPhaseAsync(CancellationToken cancellati
 					round.CoordinatorScript = GetCoordinatorScriptPreventReuse(round);
 					coinjoin = AddCoordinationFee(round, coinjoin, round.CoordinatorScript);
 
-					coinjoin = await TryAddBlameScriptAsync(round, coinjoin, allReady, round.CoordinatorScript, cancellationToken).ConfigureAwait(false);
+					var highestFeeRateTask = async () => (await Rpc.EstimateSmartFeeAsync(2, EstimateSmartFeeMode.Conservative, simulateIfRegTest: true, cancellationToken).ConfigureAwait(false)).FeeRate;
```

Comment:
This has the same problem than the #10151 and it is using the incorrect feerate. It has to use the one specified in the round.

Reply:
It is OK.